### PR TITLE
Some fixes and improvments for the README

### PR DESCRIPTION
**Pull request by jasonborg**
Some fixes and improvements for the README.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,18 +68,21 @@ working in Cloud Shell, you already have Python and `gcloud` installed.
 
 #### Set up the environment
 
-1. Run `gcloud auth login`, be sure to use the account that has permissions on
-   the Google Cloud project you are working on.
-2. Run `gcloud config set project PROJECT_ID`. Where PROJECT_ID is the project
+1. Run `gcloud config set project PROJECT_ID`. Where PROJECT_ID is the project
```
Comment (mctavish) [inline README.md:71]: This shouldn't be necessary on cloud shell; it will already be set to the current project.

File: README.md [hunk 2]
```diff
@@ -280,7 +285,9 @@ The debugger is now ready for use with your app.
     require('@google-cloud/debug-agent').start({
       useFirebase: true
```
Comment (mctavish) [inline README.md:286]: Missing comma here too.

**Review by mctavish — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (jasonborg) [reply README.md]: I've made a few changes, PTAL.

**Review by mctavish — APPROVED** [review]
Looks good!

**Pull request merged**